app/controllers/verify_controller.rb

```diff
@@ -19,7 +19,19 @@ def app_data
       base_path: idv_app_root_path,
       app_name: APP_NAME,
       completion_url: completion_url,
-      initial_values: { 'personalKey' => personal_key },
+      initial_values: {
+        'personalKey' => personal_key,
+        'firstName' => 'Bruce',
+        'lastName' => 'Wayne',
+        'address1' => '1234 Batcave',
+        'address2' => '',
+        'city'     => 'Batcavesville',
+        'state'    => 'NY',
+        'zipcode'  => '12345',
+        'dob' => '1988-03-30',
+        'ssn' => '900-12-3456',
+        'phone' => '2021234567',
```

Comment:
style-wise, I think all of these should align or none of them should align... this has some and some. We tend not to align hash items because they create lots of diff noise over time, so I'd opt for this:
```suggestion
        'personalKey' => personal_key,
        'firstName' => 'Bruce',
        'lastName' => 'Wayne',
        'address1' => '1234 Batcave',
        'address2' => '',
        'city' => 'Batcavesville',
        'state' => 'NY',
        'zipcode' => '12345',
        'dob' => '1988-03-30',
        'ssn' => '900-12-3456',
        'phone' => '2021234567',
```